agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4050,7 +4050,21 @@ bool slave_thread::handle_autoconfiguration_wsc(Socket *sd, ieee1905_1::CmduMess
         }
     }
 
-    message_com::send_cmdu(ap_manager_socket, cmdu_tx);
+    ///////////////////////////////////////////////////////////////////
+    // TODO https://github.com/prplfoundation/prplMesh/issues/797
+    //
+    // Short term solution
+    // In None EasyMesh mode, never modify hostapd configuration
+    // and in this case VAPs credentials
+    //
+    // Long term solution
+    // All EasyMesh VAPs will be stored in the platform DB.
+    // All other VAPs are manual, AKA should not be modified by prplMesh
+    ////////////////////////////////////////////////////////////////////
+    if (platform_settings.management_mode != BPL_MGMT_MODE_NOT_MULTIAP) {
+        LOG(INFO) << "EasyMesh mode - really update VAP credentials";
```

Comment:
I would rather have a message in the non-EasyMesh case... And put it at WARNING level.

Ideally even only issue the message when the SSIDs are different from the already-configured SSIDs.

Reply:
Added a warning.